py/nightwatch/qa/qprocstatus.py

```diff
@@ -0,0 +1,46 @@
+from .base import QA
+import glob
+import os
+import collections
+
+import numpy as np
+import fitsio
+import json
+
+
+from astropy.table import Table
+
+class QAQPROCStatus(QA):
+    """docstring for QANoiseCorr"""
```

Comment:
Please expand this docstring to give context about what QAPROCStatus is used for, and add other docstrings, e.g. what are the columns of the Table returned by `QAPROCStatus.run(indir)`.